### PR TITLE
Michael Valenzuela - Completed react-stopwatch-component

### PR DESCRIPTION
**Pull request by michaelvalenzuela**
stopwatch

### CODE REVIEW

File: react-stopwatch-component/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Stopwatch from './stopwatch';
+
+ReactDOM.render(
+  <Stopwatch></Stopwatch>,
```
Comment (Zhiwutian) [inline react-stopwatch-component/src/index.jsx:6]: This works, but I would change it to a JSX self closing tag.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update!

File: react-stopwatch-component/src/index.jsx (hunk 1 above)
Comment (michaelvalenzuela) [reply react-stopwatch-component/src/index.jsx:6]: Oh whoops autocomplete. Will change in a sec

**Review by scbowler — APPROVED** [review]
Looks good 👍

**Pull request merged**